src/objects/Enums/Physics.ts

```diff
@@ -0,0 +1,7 @@
+export default class keepUprightStratergies {
+  static NONE = 'NONE';
+
+  static INSTANT = 'INSTANT';
+
+  static SPRINGY = 'SPRINGY';
+}
```

Comment:
now that we have TS, this could be a real ENUM

Reply:
Changed to an enum but was throwing a bunch off errors for 'no-shadow' and 'un-used-vars', After some googling people suggested changing linting rules for no-shadow. There was also one for no-wars but I was still getting the error so I've supressed it in the enum file.
` "no-shadow": "off",
    "@typescript-eslint/no-shadow": "warn",`